Tests/SentryTests/SentryHubTests.swift

```diff
@@ -268,13 +268,13 @@ class SentryHubTests: XCTestCase {
     
     func testStartTransactionNotSamplingUsingSampleRate() {
         testSampler(expected: .no) { options in
-            options.tracesSampler = { _ in return 0.49 }
+            options.tracesSampleRate = 0.49
         }
     }
     
     func testStartTransactionSamplingUsingSampleRate() {
         testSampler(expected: .yes) { options in
-            options.tracesSampler = { _ in return 0.5 }
```

Comment:
These tests are unrelated to my changes but I fixed these to set the `tracesSampleRate` as the name of the test suggests, instead of the sampler function.